enterprise/views.py

```diff
@@ -71,10 +71,11 @@
     get_enterprise_customer_user,
     is_consent_required_for_user,
 )
-from six.moves.urllib.parse import urlencode, urljoin  # pylint: disable=import-error
+from six.moves.urllib.parse import parse_qs, urlencode, urljoin, urlsplit, urlunsplit  # pylint: disable=import-error
 
 
 logger = getLogger(__name__)  # pylint: disable=invalid-name
+EDX_ENTERPRISE_SUPPORT_URL = 'https://support.edx.org/hc/en-us/sections/115002578128-Enterprise-Learner'
```

Comment:
I believe this is a static link for all enterprises so we are fine at this stage.

If we want to have separate support links for different enterprises or if we need to update this link frequently then we should definitely make it configurable (maybe add a field in enterprise customer model). I think we can decide the solution/implementation with a new story.